wellregistry/registry/tests/admin/test_auto_complete.py

```diff
@@ -0,0 +1,62 @@
+"""
+Tests for admin.monitoring_location for Autocomplete module
+"""
+from django.contrib.admin.sites import AdminSite
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group, Permission
+from django.test import Client, TestCase
+from ...admin.monitoring_location import MonitoringLocationAdmin
+from ...models import MonitoringLocation
+class TestAutoCompleteView(TestCase):
+    # pylint: disable=too-many-instance-attributes
+    fixtures = ['test_groups.json', 'test_altitude_datum.json', 'test_counties.json',
+                'test_countries.json', 'test_horizontal_datum.json', 'test_nat_aquifer.json',
+                'test_states.json', 'test_units.json', 'test_user.json', 'test_agencies.json',
+                'test_monitoring_location.json']
+
+    def setUp(self):
+        self.superuser = get_user_model().objects.create_superuser('my_superuser')
+        self.adwr_group = Group.objects.get(name='adwr')
+        self.usgs_group = Group.objects.get(name='usgs')
+
+        self.add_permission = Permission.objects.get(codename='add_monitoringlocation')
+        self.view_permission = Permission.objects.get(codename='view_monitoringlocation')
+        self.change_permission = Permission.objects.get(codename='change_monitoringlocation')
+        self.delete_permission = Permission.objects.get(codename='delete_monitoringlocation')
+
+        self.adwr_user = get_user_model().objects.create_user('adwruser')
+        self.adwr_user.groups.add(self.adwr_group)
+        self.adwr_user.user_permissions.set([
+            self.add_permission,
+            self.view_permission,
+            self.change_permission,
+            self.delete_permission
+        ])
+        self.adwr_user.is_staff = True
+        self.adwr_user.save()
+
+        self.usgs_user = get_user_model().objects.create_user('usgsuser')
+        self.usgs_user.groups.add(self.usgs_group)
+        self.usgs_user.user_permissions.set([
+            self.add_permission,
+            self.view_permission,
+            self.change_permission,
+            self.delete_permission
+        ])
+        self.usgs_user.is_staff = True
+        self.usgs_user.save()
+
+        self.site = AdminSite()
```

Comment:
You also don't need these two lines. Please remove them.

Reply:
fixed